.github/workflows/ci.yml

```diff
@@ -286,7 +294,7 @@ jobs:
           steps.python-changes.outputs.run == 'true'
           && startsWith(matrix.os, 'macos-') != true
         run: poetry run pytest ${{ env.pytest-base-args }} tests --runmountpoint --runslow --rungui -m gui
-        timeout-minutes: 10
+        timeout-minutes: 20
```

Comment:
Must investigate why the the openssl-based implementation is much slower than the RustCrypto one (or than the previous Python one that was itself using openssl under the hood !)

Reply:
The reason is that openssl-based implemention with `opt-1` profile is as slow as `opt-0` (`10min` vs `11min` with my machine) while with rustcrypto, this is much better (`4min` vs `11min`)